webui/react/src/components/InteractiveTable.tsx

```diff
@@ -367,17 +362,15 @@ const InteractiveTable: InteractiveTable = ({
     pageWidth ]);
 
   const handleChange = useCallback(
-    /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-    (tablePagination: any, tableFilters: any, tableSorter: any): void => {
+    (tablePagination, tableFilters, tableSorter): void => {
```

Comment:
question: You should be able to pull the types from the antd Table onChange, right? In which case tablePagination is `TablePaginationConfig`, tableFilters is `Record<string | FilterValue | null>`, and tableSorter is `SorterResult<RecordType> | SorterResult<RecordType>[]`

Reply:
This is a situation where we would need to have `RecordType` parameter available in the function, like with the above. Will revisit.